core/test-runtime/src/lib.rs

```diff
@@ -77,6 +78,17 @@ pub struct Transfer {
 	pub nonce: u64,
 }
 
+impl Transfer {
+	/// Convert into a signed extrinsic.
+	#[cfg(feature = "std")]
+	pub fn into_signed_tx(self) -> Extrinsic {
+		let signature = keyring::Keyring::from_raw_public(
```

Comment:
should just be `keyring::Keyring::from_public(self.from)`. If you're type-running something's wrong.